IFC4x3/Sections/Resource definition data schemas/Schemas/IfcGeometricModelResource/Entities/IfcSectionedSurface/Documentation.md

```diff
@@ -1,15 +1,16 @@
-A kind of surface constructed by sweeping potentially varying open cross sections along a curve horizontally (or near horizontally). The surface is generated by sweeping the CrossSections between CrossSectionPositions; linear interpolation is assumed, unless transitions curves between cross section points are indicated by OpenCrossProfileDef.Tags.
-The CrossSections are oriented with the Y axis of each profile facing upwards in +Z direction or vertically perpendicular to the Directrix, depending on the Attribute FixedAxisVertical value.
+A surface constructed by sweeping potentially varying open cross sections along a curve horizontally (or near horizontally). The surface is generated by sweeping the CrossSections between CrossSectionPositions; linear interpolation is assumed, unless transitions curves between cross section points are indicated by OpenCrossProfileDef.Tags.
+
+The profile normal is derived from the associated _IfcAxis2PlacementLinear_, not necessarily the tangent of the _Directrix_. The profile X axis is oriented perpendicularly to the left of the _Directrix_ (same direction as _RefDirection_ derived in _IfcAxis2PlacementLinear_) as facing forward along the directrix, and the profile Y axis is oriented upwards according to _Axis_ attribute of _IfcAxis2PlacementLinear_.
+
+Figure 1 illustrates an _IfcSectionedSurface_ defined using _IfcOpenCrossProfileDef_
+
 !["sectioned surfacequot;](../../../../../../figures/IfcSectionedSurface_1.png "Figure 1 &mdash; Sectioned surface using _IfcOpenCrossProfileDef_")
 
 In case of branching longitudinal breaklines, the SectionedSurface may use OpenCrossProfileDef instances with varying number of cross section points. In that case, the point in two consecutive cross sections that are connected are identified by the same tag value.
```

Comment:
```suggestion
In case of branching longitudinal breaklines, the _SectionedSurface_ may use _OpenCrossProfileDef_ instances with varying number of cross section points. In that case, the point in two consecutive cross sections that are connected are identified by the same tag value.
```